src/odemis/driver/tescan.py

```diff
@@ -1158,6 +1222,7 @@ def _doMoveAbs(self, pos: dict):
 
             logging.debug("Stage move to %s completed", pos)
             self._updatePosition()
+            self._checkPosition(pos_before_move, pos, rel=False)
```

Comment:
Probably the most important part of this whole change is missing: explain *why* this is needed. If I understand correctly, that's because the stage might fail to move, but there is no way to detect it explicitly, so you need to detect a failure to move by just comparing the requested position to the final position? Or is it also like on the TFS stage, where it can report it's finished moving, but it's not?

Reply:
Added to the comment, It is first option that we need to check explicitly that the stage has moved to the requested position as it does not give any error when the stage fails to move.